src/rust/engine/process_execution/src/remote_cache.rs

```diff
@@ -251,7 +304,7 @@ impl CommandRunner {
     directory
       .files
       .iter()
-      .find(|n| n.get_name() == file_base_name)
+      .find(|n| n.name == file_base_name.to_string_lossy())
```

Comment:
Is there a better way to convert `OSStr` to `String` in path-name situations?

Reply:
We can do the comparison in the other direction - `String`s are trivially convertable to `OsStr{,ing}`s, so if we  convert the first rather than the second it should be good

Reply:
Will fix.